server/src/test/java/io/crate/integrationtests/PartitionedTableConcurrentIntegrationTest.java

```diff
@@ -24,6 +24,7 @@
 import static com.carrotsearch.randomizedtesting.RandomizedTest.$;
 import static com.carrotsearch.randomizedtesting.RandomizedTest.randomAsciiLettersOfLength;
 import static io.crate.testing.Asserts.assertThat;
+import static org.assertj.core.api.Assertions.assertThat;
```

Comment:
minor, but since we import our assert you can remove.

Reply:
eclipse.jdt.ls for some reason insists on adding them.

Reply:
no worries, super minor.